src/RoboSaverVirtualModule.sol

```diff
@@ -41,6 +46,11 @@ contract RoboSaverVirtualModule is
     IERC20 immutable STEUR;
     IERC20 immutable EURE;
 
+    IRewardPoolDepositWrapper constant AURA_DEPOSITOR =
+        IRewardPoolDepositWrapper(0x0Fec3d212BcC29eF3E505B555D7a7343DF0B7F76);
+    IBoosterLite constant AURA_BOOSTER = IBoosterLite(0x98Ef32edd24e2c92525E59afc4475C1242a30184);
+    IVoterProxyLite constant AURA_PROXY = IVoterProxyLite(0xC181Edc719480bd089b94647c2Dc504e2700a2B0);
```

Comment:
realised this is not used anywhere. i removed it on #70. lmk if indeed there is plan to use it at all

Reply:
nice spot, removed